tap-snapshots/test-unit-ts.js-TAP.test.js

```diff
@@ -26,14 +26,40 @@ Object {
 }
 `
 
-exports[`test/unit/ts.js TAP TS Unit Tests > TS Build Tree contains correct output source 1`] = `
+exports[`test/unit/ts.js TAP TS Unit Tests > [TS Build - Bare Imports] Correct output source code 1`] = `
+Array [
+  "import * as ts from 'typescript';\\nconsole.log(\`Script Target: \${ts.ScriptTarget.ES2020}\`);\\n\\nconsole.log('This line was added by a post processor!!');",
+]
+`
+
+exports[`test/unit/ts.js TAP TS Unit Tests > [TS Build - Bare Imports] Correct output tree file names 1`] = `
+Array [
+  "file:///{fs}/presm/dist/test/fixtures/ts-build-bare/main.mjs",
+]
+`
+
+exports[`test/unit/ts.js TAP TS Unit Tests > [TS Build - Relative Imports] Correct output source code 1`] = `
+Array [
+  "import mymodule, { numArr } from \\"/{fs}/presm/test/fixtures/ts-build-relative-imports/mymodule.ts\\";\\nconsole.log(\`Module Name: \${mymodule}\`);\\nconst myNumArr = numArr(5);\\nconsole.log(\`My Num Arr: \${myNumArr}\`);\\n\\nconsole.log('This line was added by a post processor!!');",
+  "export default function defaultFunction() {\\n    const returnStringArr = ['MyModule', 'MyModule', 'MyModule'];\\n    return returnStringArr;\\n}\\nexport function numArr(num) {\\n    const arr = [];\\n    let c = 0;\\n    while (c < num) {\\n        arr.push(c);\\n        c++;\\n    }\\n    return arr;\\n}\\n\\nconsole.log('This line was added by a post processor!!');",
```

Comment:
It's getting kind of hard to review this because code with escaped lines is pretty awkward. Would it be possible to turn this into two snapshot tests so it's not wrapped in `Array`, assuming that a string would be printed directly..? Alternatively, it may be better to put the expected output into "real" files and not use the snapshot matching.

Reply:
Sure let me look into this

Reply:
Snapshotting individual files instead of in an an array fixed readability issues - good call!  Please see new snapshot file